tests/Fixtures/Entity/Foo.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Zenstruck\Foundry\Tests\Fixtures\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="foos")
+ */
+class Foo
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\OneToOne(targetEntity=Bar::class, cascade={"persist", "remove"})
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $oneToOne;
+
+    /**
+     * @ORM\OneToMany(targetEntity=Bar::class, mappedBy="foo")
+     */
+    private $oneToMany;
+
+    /**
+     * @ORM\ManyToOne(targetEntity=Bar::class)
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $manyToOne;
+
+    /**
+     * @ORM\ManyToMany(targetEntity=Bar::class)
+     */
+    private $manyToMany;
+
+    public function __construct()
```

Comment:
I think we can remove the boilerplate below - we aren't actually creating these entities.